source/boards/navigate-boards.rst

```diff
@@ -4,6 +4,11 @@ Navigate boards
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
   
+.. |options-icon| image:: ../images/dots-horizontal_F01D8.svg
+  :height: 24px
```

Comment:
You can safely remove the height/width values for compass design elements. The default size of these images works well for the docs.